pkg/apis/istio/v1alpha1/istioinstaller_types.proto

```diff
@@ -0,0 +1,585 @@
+syntax = "proto3";
+
+package v1alpha1;
+
+import "k8s.io/api/core/v1/generated.proto";
+import "k8s.io/api/autoscaling/v1/generated.proto";
+import "k8s.io/apimachinery/pkg/apis/meta/v1/generated.proto";
+import "github.com/gogo/protobuf/protobuf/google/protobuf/wrappers.proto";
+
+/*
+IstioInstall is a schema for both defining and overlaying Istio control plane installations.
+The Istio operator uses a compiled-in InstallSpec as default installation values, along with compiled-in install charts.
+Running the operator with an empty user defined InstallSpec results in an installation with default values, using the
+default charts.
+
+The simplest install specialization is to point the user InstallSpec baseValuesPath to a different values file, for
+example an Istio auth installation, which will use the values associated with the auth installation profile for Istio.
```

Comment:
Can I assume I can use a service profile here?  Nit to help user easier understand this - I think it makes sense to refer to one of our supported profile like mTLS strictly enforced profile or minimal profile.

Reply:
Reading it closer, I think that is your intention, it just was not aligned well with our terminologies:  https://istio.io/docs/setup/kubernetes/additional-setup/config-profiles/